tests/snapshots/tests.contrib.openai.test_openai_v0.test_integration_service_name[None-None].json

```diff
@@ -8,12 +8,11 @@
     "parent_id": 0,
     "meta": {
       "_dd.p.dm": "-0",
-      "_dd.p.tid": "654a694400000000",
+      "_dd.p.tid": "65674d7100000000",
       "component": "openai",
       "language": "python",
       "openai.api_base": "https://api.openai.com/v1",
       "openai.api_type": "open_ai",
-      "openai.base_url": "https://api.openai.com/v1/",
```

Comment:
Why is base_url getting deleted with no replacement here? Is it related to the resource name change or something else?

Reply:
I noticed that in some cases `base_url` is missing, and not replaced, and in some cases it’s `api_base` or `api_type` .  Looking at the code though, and the Openai versions for which we don’t have certain tags, these seem to be correct…. e.g.[ we’re missing api_base and api_type here using >openai==1.0.0](https://github.com/DataDog/dd-trace-py/pull/7778/files#diff-bca930a237a23fc479cbd7a4a881a0afc62599f18a2bb23890e43a699969c613) which makes sense given the code here: https://github.com/DataDog/dd-trace-py/blob/2.x/ddtrace/contrib/openai/patch.py#L185-L197
So… why didn’t we catch this earlier?

Reply:
@Yun-Kim

Reply:
Discussed offline, I overlooked this when creating snapshots for v0/v1 - we reuse the same snapshots for most tests for v0/v1 (which is why we added `base_url/api_type/api_base` tags to all snapshots) but not for `test_integration_service_name` 🤦 so we can go ahead with these changes, thanks!